webapp/locales/de.json

```diff
@@ -102,7 +102,7 @@
       "projects": "Projekte",
       "invites": "Einladungen",
       "follows": "Folgen",
-      "shouts": "Shouts"
+      "shouts": "Empfehlungen"
```

Comment:
Actually, I think I :heart it. Indeed, shouting might have a negative connotation in English.

Shouting in Human Connection means to increase the reach of the post. Let more people see it. So I kind of like your translation of it! :+1:

Reply:
yes, we have a lot of expressions in English like `to shout something from the rooftop`, `to give a shout-out`, etc..

![Screenshot at 2019-06-13 21:02:41](https://user-images.githubusercontent.com/26943915/59474819-af234c80-8e1e-11e9-8913-5c29979f6eff.png)
![Screenshot at 2019-06-13 21:02:29](https://user-images.githubusercontent.com/26943915/59474825-b2b6d380-8e1e-11e9-8463-5d9fdb3e6e2c.png)